tests/integration/it_utils.py

```diff
@@ -154,14 +154,14 @@ async def _run_async_test(self, client):
 
         def modified_test(self):
             if not websockets_only:
-                with self.subTest(version="sync", client="json"):
-                    _run_sync_test(self, JSON_RPC_CLIENT)
                 with self.subTest(version="async", client="json"):
                     asyncio.run(_run_async_test(self, ASYNC_JSON_RPC_CLIENT))
-            with self.subTest(version="sync", client="websocket"):
-                _run_sync_test(self, WEBSOCKET_CLIENT)
+                with self.subTest(version="sync", client="json"):
+                    _run_sync_test(self, JSON_RPC_CLIENT)
```

Comment:
why was thsi moved? just an accident?

Reply:
It makes debugging easier - all async stuff is run before the sync stuff, and async stuff has a better stack trace.